Brocfile.js

```diff
@@ -21,7 +21,10 @@ var assetsWithTemplates = handlebars('assets', {
 var appJs = concat(assetsWithTemplates, {
   inputFiles: [
     'js/setup.js',
-    'templates/**/*.js'
+    'templates/**/*.js',
+    'js/blog-mvc.js',
+    'js/blog.js',
```

Comment:
Order is important here... You are getting an error `CommentList is not a function` because you put your `js/blog-mvc.js` before everything... But it needs the `blog.js` and `views/app.js` file.